recipes/protobuf/all/patches/upstream-pr-7288-android-log.patch

```diff
@@ -0,0 +1,25 @@
+--- a/cmake/libprotobuf.cmake
++++ b/cmake/libprotobuf.cmake
+@@ -121,6 +121,9 @@ endif()
+ if(protobuf_LINK_LIBATOMIC)
+   target_link_libraries(libprotobuf atomic)
+ endif()
++if(${CMAKE_SYSTEM_NAME} STREQUAL "Android")
++	target_link_libraries(libprotobuf log)
++endif()
```

Comment:
Patch like this could be implemented not like a patch, but as a small change in this file:
https://github.com/conan-io/conan-center-index/blob/master/recipes/protobuf/all/CMakeLists.txt

This will work forever and easy to maintain for future versions.

Reply:
@uilianries @prince-chrismc please, share your opinion on this.

Reply:
I general I would agree but since this PR is already merge (v3.14.0) the future maintenance should not suffer.
https://github.com/protocolbuffers/protobuf/pull/7288

Reply:
I agree with @mathbunnyru , but as it's a patch related for building support and already accepted by the upstream, I see no problem in this case.

Reply:
There's also the complexity of adding protobuf versions which (imo) is always more complicated than it seems.
This case is acceptable. It follow our usually pattern.

Reply:
Thanks, everyone. I suggest we add upstream PR link to the patch and it should be good to go.